app/views/admin/words/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for [:admin, @category, @word] do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+
+    <div class="form-group">
+        <%= f.label :word %>
+        <%= f.text_field :content, class: 'form-control' %>
+    </div>
+
+    <%= f.fields_for :answers do |a| %>
+    
+        <div class="form-group">
+            <%= a.label :option %>
+            <div class="row">
+                <div class="col-md-11">
+                    <%= a.text_field :content, class: 'form-control' %>
+                </div>
+
+                <div class="col-md-1">
+                <%= a.check_box :correct %>
+                </div>
+            </div>
+        </div>
+
+    <% end %>
+
+    <%= f.submit class: "form-control btn btn-primary" %>
+
+<% end %>
+
+<script>
+    $('#new_word input[type=checkbox]').change(function() {
+       $('#new_word input[type=checkbox]:checked').not(this).prop('checked', false);
+    });
```

Comment:
@shokogyu Since this javascript is now being used for both edit & new forms, please update `#new_word` to `.word_form`.

And then add a `word_form` class in the `form_for` above.